src/function.lisp

```diff
@@ -44,6 +44,38 @@
              (and result-type
                   (list (format nil "~a *result" (c-type result-type))))))))
 
+(defun c-function-definition (name result-type typed-lambda-list
+                              &key (function-prefix "")
```

Comment:
Docstring or a kindness-to-future-maintainers comment

Reply:
Docstring added